src/PrestaShopBundle/Twig/Extension/JsRouterMetadataExtension.php

```diff
@@ -85,7 +85,7 @@ public function getJsRouterMetadata()
     {
         return [
             // base url for javascript router
-            'base_url' => $this->requestStack->getCurrentRequest()->getBaseUrl(),
+            'base_url' => $this->requestStack->getCurrentRequest() ? $this->requestStack->getCurrentRequest()->getBaseUrl() : '',
```

Comment:
Is this fix needed now that you handle this in the AdminController? Maybe it's a remnant of a previous implementation that didn't work as expected?

Reply:
Well i no longer use this so i guess its not needed but still there is possibility that getCurrentRequest() will return null and if that heppens the shop gets exception if we try to call getBaseUri. For example when I tried using this on some cases shop would throw exception. But if you think its better to leave the old way i can change it back